torch/csrc/distributed/rpc/init.cpp

```diff
@@ -214,6 +214,78 @@ PyObject* rpc_init(PyObject* /* unused */) {
                   If the current node is the owner, returns a reference to the
                   local value. Otherwise, throws an exception.
               )")
+          .def(
+              "rpc_sync",
+              [&](PyRRef& self) {
+                  return self.createRRefProxy(self, RRefProxyType::RPC_SYNC);
+              },
+              py::call_guard<py::gil_scoped_release>(),
+              R"(
+                  Create a helper proxy to easily launch an ``rpc_sync`` using
+                  the owner of the RRef as the destination to run functions on
```

Comment:
Wondering, is there a use case to eventually support these APIs on non-RRef owner nodes?

Reply:
good point! we could let this function take a `dst` arg to support that.

Reply:
If the plan is to implicitly pull the object to `dst` and run the method, we should be cautious 
 with the API. In pyper use case, we actually add extra assert to make sure code is always on its owner.

Reply:
I see. Then, let's not add that to this PR. We can add it later if we see real use cases and request for it.